commands/show-bl-hostname.go

```diff
@@ -111,65 +125,25 @@ func showBLHostnamesHuman(hostnames []hostname.AnalysisView) error {
 			strconv.Itoa(entry.TotalBytes),
 		}
 
-		sort.Strings(entry.ConnectedHosts)
-		serialized = append(serialized, strings.Join(entry.ConnectedHosts, " "))
+		var sourceIPs []string
```

Comment:
The logic here appears to essentially be the same as what is used in the showBLIPs function in the show-bl-ip.go function (around line 159).  The logic here uses an if/else with a for loop in each branch.  The logic in showBLIPs uses a single for loop with an if/else statement inside.  The former is likely faster as it doesn't need to evaluate the if/else at each loop.  The latter is only marginally more concise, imo...so I would lean towards using the former for consistency and a small speed boost. Your call though and not a big deal.

Reply:
Changed to match showBLIPs for consistency. Since this is printing logic, the speed shouldn't matter too much given it has to aggregate over the database first. However, I like keeping the code patterned and predictable.